### PR TITLE
Feature 57 set method for send email when user report test or symptoms

### PR DESCRIPTION
**Pull request by fego-dev**
# Description
I refactor the create method for incident, now the user should not send the user id and the current day, also i created new method for send email when  user report test or symptoms

### CODE REVIEW

**Review by OchoaHub — APPROVED** [review]
WOW! Nice code bro!

File: app/controllers/api/v1/incidents_controller.rb [hunk 1]
```diff
@@ -36,4 +38,14 @@ def incident_params
...
+    def set_aditional_data
+      @incident.user_id = current_user.id
+      @incident.date_incident = Time.now
+    end
+
+    def alert_mail
+      to = PlaceHistory.possible_infections(@incident.user_id).pluck("email")
+      NotificationMailer.alert_test(to)
+    end
+
```
Comment (MiguelOcegueraM) [inline app/controllers/api/v1/incidents_controller.rb:50]: nice method mate

**Review by LPMendoza — APPROVED** [review]
Good job man it's very clear

**Review by MiguelOcegueraM — APPROVED** [review]
the code is very clean bro

**Review by jnegrete15 — APPROVED** [review]

**Pull request merged**